auth/src/main/java/com/firebase/ui/auth/ui/FragmentBase.java

```diff
@@ -1,28 +1,49 @@
 package com.firebase.ui.auth.ui;
 
 import android.content.Intent;
+import android.content.IntentSender;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.annotation.RestrictTo;
 import android.support.v4.app.Fragment;
+import android.view.ContextThemeWrapper;
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 public class FragmentBase extends Fragment {
-    protected FragmentHelper mHelper;
+
+    protected FlowParameters mFlowParameters;
```

Comment:
Shouldn't the flow params be private since we have a getter to handle initialization?